decidim-budgets/config/locales/en.yml

```diff
@@ -66,6 +66,12 @@ en:
         project:
           fields:
             title: Title
+      order_summary_mailer:
+        order_summary:
+          selected_projects: 'The projects that you have selected are:'
+          subject: You have voted on the %{space_name} participatory space
+          voted_on_space: You have voted on the budets for the %{space_name} participatory space.
+          voted_on_space_with_scope: You have voted on the budets for the %{space_name} participatory space on the %{scope_name} district.
```

Comment:
The scope could be of another type, maybe we should use the scope_type name?
Also, you have a typo with "budgets" in the lines 73 and 74.